pkg/apis/core/validation/validation.go

```diff
@@ -8167,6 +8167,18 @@ func ValidateResourceQuotaStatusUpdate(newResourceQuota, oldResourceQuota *core.
 // ValidateNamespace tests if required fields are set.
 func ValidateNamespace(namespace *core.Namespace) field.ErrorList {
 	allErrs := ValidateObjectMeta(&namespace.ObjectMeta, false, ValidateNamespaceName, field.NewPath("metadata"))
```

Comment:
In following the logic for ValidateNamespaceName it goes:

ValidateNamespaceName -> apimachineryvalidation.ValidateNamespaceName -> NameIsDNSLabel -> validation.IsDNS1123Label(name)

This maps to "ShortName" for declarative validation (not long name):
```
func ShortName[T ~string](_ context.Context, op operation.Operation, fldPath *field.Path, value, _ *T) field.ErrorList {
	if value == nil {
		return nil
	}
	var allErrs field.ErrorList
	for _, msg := range content.IsDNS1123Label((string)(*value)) {
		allErrs = append(allErrs, field.Invalid(fldPath, *value, msg).WithOrigin("format=k8s-short-name"))
	}
	return allErrs
}
```